docs/changelog.rst

```diff
@@ -11,6 +11,21 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+v3.24.3 (2021-08-21)
+Bugfixes
```

Comment:
@gaborbernat this happened again: no underline and an empty line. You may want to consider downgrading Towncrier or updating the release scripts...

cc @asottile WDYT?